src/index.js

```diff
@@ -1,58 +1,55 @@
 
 /**
- * See `http://www.irs.gov/Individuals/General-ITIN-Information` for more information.
- *
  * Social Security number (SSN) is a nine-digit number issued to U.S. citizens, permanent residents,
  * and * temporary (working) residents under section 205(c)(2) of the Social Security Act, codified
  * as 42 U.S.C. 405(c)(2).
+ *
+ * See `http://www.irs.gov/Individuals/General-ITIN-Information` for more information.
  */
 
 /**
- * Module dependencies.
+ * Blacklists.
  */
 
-import _ from 'lodash';
+const blacklists = {
+  soft: ['078-051-120', '219-099-999', '457-555-462'],
+  strict: ['078051120', '219099999', '457555462']
+};
 
 /**
- * Blacklist.
+ * Expressions.
  */
 
-const blacklist = [
-  '078051120',
-  '219099999',
-  '457555462'
-];
-const expression = /^(?!666|000|9\d{2})\d{3}(?!00)\d{2}(?!0{4})\d{4}$/;
+const expressions = {
+  soft: /^(?!666|000|9\d{2})\d{3}-(?!00)\d{2}-(?!0{4})\d{4}$/,
+  strict: /^(?!666|000|9\d{2})\d{3}(?!00)\d{2}(?!0{4})\d{4}$/
```

Comment:
I don't think we need to add another expression for the soft case. Simply replacing the characters we would like to support (e.g. multiple spaces and dashes?) and then running the resulting value through the normal expression would suffice in my opinion.